linux_os/guide/services/ntp/chronyd_client_only/rule.yml

```diff
@@ -15,8 +15,6 @@ rationale: |-
 
 severity: unknown
 
-platform: machine  # The check uses service_... extended definition, which doesnt support offline mode
```

Comment:
Containers have packages and services too. Reverting this change.

Reply:
ditto (see `service_cockpit_disabled`)